package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "axios": "0.17.1",
-    "debug": "2.6.8",
+    "debug": "2.6.9",
```

Comment:
That is why operators are useful. If this was `~2.6.8` or `^2.6.9` this would have been fixed for all people that depend to a package that depends on localtunnel...

Reply:
Wouldn't it be better to use `^3.1.0` instead?